dm/master/server.go

```diff
@@ -711,7 +711,7 @@ func (s *Server) QueryStatus(ctx context.Context, req *pb.QueryStatusListRequest
 }
 
 // adjust unsynced field in sync status by looking at DDL locks.
-// because if a DM-worker doesn't receive any shard DDL, it doesn't even know it's unsynced for itself
+// because if a DM-worker doesn't receive any shard DDL, it doesn't even know it's unsynced for itself.
```

Comment:
why didn't CI discover there's a dot missing 🤔

Reply:
I am also curious, let me cc

Reply:
oh it's because we open [fix: true](https://github.com/pingcap/dm/blob/4c1857c3ff4ee456a622dc0a9a05fe7b3d9e43d4/.golangci.yml#L119) in golangci , as a result golangci wiil auto fix the issue. 

i wil fix this.

Reply:
Can we use set it true as default and false for [check-and-build](https://github.com/pingcap/dm/blob/master/.github/workflows/check-and-build.yml) in CI? So we can fix it automatically by `make build`, and discover it by GitHub action